packages/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerPlugin.java

```diff
@@ -207,6 +209,87 @@ private File scaleImage(Image image, Double maxWidth, Double maxHeight) throws I
     fileOutput.write(outputStream.toByteArray());
     fileOutput.close();
 
+    if (shouldDownscale) copyExif(image.getPath(), scaledCopyPath);
+
     return imageFile;
   }
+
+  private void copyExif(String filePathOri, String filePathDest) {
+    try {
+      ExifInterface oldexif = new ExifInterface(filePathOri);
+      ExifInterface newexif = new ExifInterface(filePathDest);
+
+      int build = Build.VERSION.SDK_INT;
+
+      // From API 11
+      if (build >= 11) {
+        if (oldexif.getAttribute("FNumber") != null) {
+          newexif.setAttribute("FNumber", oldexif.getAttribute("FNumber"));
+        }
+        if (oldexif.getAttribute("ExposureTime") != null) {
+          newexif.setAttribute("ExposureTime", oldexif.getAttribute("ExposureTime"));
+        }
+        if (oldexif.getAttribute("ISOSpeedRatings") != null) {
+          newexif.setAttribute("ISOSpeedRatings", oldexif.getAttribute("ISOSpeedRatings"));
+        }
+      }
+      // From API 9
+      if (build >= 9) {
+        if (oldexif.getAttribute("GPSAltitude") != null) {
+          newexif.setAttribute("GPSAltitude", oldexif.getAttribute("GPSAltitude"));
+        }
+        if (oldexif.getAttribute("GPSAltitudeRef") != null) {
+          newexif.setAttribute("GPSAltitudeRef", oldexif.getAttribute("GPSAltitudeRef"));
+        }
+      }
+      // From API 8
+      if (build >= 8) {
+        if (oldexif.getAttribute("FocalLength") != null) {
+          newexif.setAttribute("FocalLength", oldexif.getAttribute("FocalLength"));
+        }
+        if (oldexif.getAttribute("GPSDateStamp") != null) {
+          newexif.setAttribute("GPSDateStamp", oldexif.getAttribute("GPSDateStamp"));
+        }
+        if (oldexif.getAttribute("GPSProcessingMethod") != null) {
+          newexif.setAttribute("GPSProcessingMethod", oldexif.getAttribute("GPSProcessingMethod"));
+        }
+        if (oldexif.getAttribute("GPSTimeStamp") != null) {
+          newexif.setAttribute("GPSTimeStamp", "" + oldexif.getAttribute("GPSTimeStamp"));
+        }
+      }
+      if (oldexif.getAttribute("DateTime") != null) {
```

Comment:
Suggestion:
How about writing a little private function to set all of these?

private void setIfNotNull(String property) {
   if (oldexif.getAttribute(property) != null) {
      newexif.setAttribute(property, oldexif.getAttribute(property));
   }
}
...
list of properties to instantiate:
List<String> attributes = Arrays.asList("DateTime", "Flash", ....);
for (String attribute : attributes) {
   setIfNotNull(attribute);
}